lib/vsc/install/shared_setup.py

```diff
@@ -1201,18 +1201,28 @@ def sanitize(name):
             return ",".join([klass.sanitize(r) for r in name])
 
         else:
-            if os.environ.get('VSC_RPM_PYTHON', 'NOT_ONE') == '1':
+            pyvers = os.environ.get('VSC_RPM_PYTHON', None)
+            if pyvers in ("1", "2", "3"):
+                if pyvers == '1':
+                    pyvers = ''
+
                 # hardcoded prefix map
                 for pydep, rpmname in PYTHON_BDIST_RPM_PREFIX_MAP.items():
                     if name.startswith(pydep):
-                        return rpmname+name[len(pydep):]
+                        newname = (rpmname+name[len(pydep):]) % pyvers
+                        log.debug("new sanitized name %s from map (old %s)", newname, name)
+                        return newname
 
                 # more sensible map
                 p_p = (not ([x for x in NO_PREFIX_PYTHON_BDIST_RPM if name.startswith(x)] or
```

Comment:
please rename `p_p` to something more sensible while you're at it

`is_python_pkg`?